README

```diff
@@ -1,17 +1,21 @@
 Term::EditLine
 ==============
 
-Term::EditLine is a object oriented perl interface to the NetBSD editline
-library. To use the module you will have to install this library. Read
+Term::EditLine is an object oriented perl interface to the NetBSD editline
+library. To use the module you will have to install this library first. Read
```

Comment:
This copy isn't accurate anymore, if the `libedit` isn't available then `Alien::Editline` will be used to download and build the library for you.

Reply:
I just added "first". Do you mean that sentence should be eliminated altogether?

Reply:
I'd like to reword it to represent how it works now.  Something like "This module will fallback on Alien::Editline if your system does not have libedit already installed".